Source/WebGPU/WebGPU/CommandEncoder.swift

```diff
@@ -25,6 +25,7 @@
 
 import Metal
 import WebGPU_Internal
+import wtf.Cxx.text.TextStream
```

Comment:
I'm unconvinced that we want `Cxx` here (and in all the thousands of future usages). Can you get rid of the `explicit`?